### ISSUE TITLE
Write GAL equations for decoding logic

### ISSUE THREAD

**Issue by apoloval** [apoloval/artemisa#142]
We to replace most of AND, OR and NOT gates by a single 22V10 GAL. 

The goal of this issue is to have a method to describe the logic equations and generate the JEDEC files from it. The alternatives are:

- Use WinCUPL. The official method recommended by Atmel. Only Windows, ancient tool, plenty of bugs. 
- Use [GALasm](https://github.com/daveho/GALasm). Community tool, poor documentation, really uncertain.

**Comment by apoloval** [comment]
Finally I used GALasm tool. I found a bug, fixed and submitted [the corresponding PR](https://github.com/daveho/GALasm/pull/6). Apart from that, it looks promising. 

All the logic found in the decoding sheet has been implemented, with the following exceptions or particularities:

- `/CS1`, `/CS2` and `/CS12` are generated by the GAL. The decoder U15B is no longer needed. 
- The equations of PSG selection signals have been simplified. In this version we use more product terms instead of reusing `PSGBDIR` in `PSGBC1`. 
- The decoder U26 is still used to generate the select signals for peripherals. In order to remove U10D and U13A gates, the enable inputs of the decoder has been altered. A new signal `/IOSL` is generated by the GAL, indicating an IO operation is requested. This excludes the activations of `/IORQ` for vectors and instructions in interrupt modes 0 and 2. This signal can be directly connected to the selection of U26, along A7 and A6 to generate IO select signals for specific devices. 
- The `/PPIRDY` signal is still produced by U17B. There are no enough outputs for it in the GAL.

Done in 26440c6caba0170a7820a9408d4b950223edcfe9.
